### DISCUSSION TITLE
Easy way to integrate TailwindCSS and Postcss with Remix Project

### DISCUSSION THREAD

**Discussion by AjayTheWizard** [remix-run/remix#4361] [Show and tell / tips]
I found another way to setup TailwindCSS or Postcss for Remix Project which is so easy to setup rather than working with `npm-run-all` which makes DX.
Here the name of package `remix-postcss`. May be You could introduce in Docs.

## Installation

```
npm i -D tailwindcss postcss remix-postcss some-postcss-plugin
```
Then create `postcss.config.js` and add configuration at root folder of the project and edit as follows in
`app/server.entry.ts`

```
import { remixPostcss } from "remix-postcss";

...
export default function handleRequest(
  request: Request,
  responseStatusCode: number,
  responseHeaders: Headers,
  remixContext: EntryContext
) {
  remixPostcss(remixContext);
...
```

**Comment by kiliman** [comment]
Hmm... not sure why you'd want to make postcss, a **build-time** process, into a **run-time** process. This essentially rebuilds your CSS on every document request.
